canonical/ubuntu.com
# Build /managed/firefighting-support page

## Done

[Design](https://www.figma.com/file/elTbjLdEYglzrRX01EvGd2/24.04-U.com---Firefighting-Support?type=design&node-id=40-1615&mode=design&t=iuq6Qzen1bUXU2Fd-0)
[Copydoc](https://docs.google.com/document/d/1sWNMmLoyOFUI9qZE3fqPhgZWJSpb-e6-juGrW7A5rr0/edit)

## QA

- Go to https://ubuntu-com-13845.demos.haus/managed/firefighting-support
- Compare content against [Copydoc](https://docs.google.com/document/d/1sWNMmLoyOFUI9qZE3fqPhgZWJSpb-e6-juGrW7A5rr0/edit)
- Check on different screen sizes

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-9794


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
